config/custom_items.txt

```diff
@@ -933,3 +933,13 @@ item_icon: pwcd
 item_desc: This wrist mounted computer was the Belters' answer for the older PDAs at the time, seeing use throughout the golden era when Ceres still operated as a mining facility. It now still sees wide use on various civilian to government vessels by use of engineering, or mining departments to the rare science department. Having a strong reputation for being modular, this one comes with an alarm clock and radio but the geiger counter seems to have been busted due to time. 
 req_titles: Engineer, Prospector, Bartender, Crewman, Passenger, Corporate Liaison, Loss Prevention Associate, Senior Engineer, Roboticist, Chief Engineer, Biomechanical Engineer 
 }
+{
+ckey: rodmuth
+character_name: Degbert Kentbritt
+item_path: /obj/item/clothing/suit/storage/toggle/bomber
+item_name: Expensive Suit
+item_icon: expensive_suit
+item_desc: This suit is handcrafted by one of the finest tailor shops in Selene. Designed using an old silk weave pattern found on Luna, this jacket is that of legend. Being adorned, with golden buttons polished to a glossy finish. If one were to look closely at the buttons they would find little ornate designs of the Coat of arms of Queensland on them.
```

Comment:
`This suit was handcrafted by one of the finest tailor shops on Selene. Designed using an old silk weave pattern found on Luna, this jacket the stuff of legends. It is adorned with golden buttons, polished to a glossy finish. If one were to look closely at the buttons, they would find little ornate designs of the Coat of arms of Queensland.`